### ISSUE TITLE
Should the NSAccessibility scroll-bar API be included in mapping for scrollbar role?

### ISSUE THREAD

**Issue by joanmarie** [w3c/core-aam#242]
NSAccessibility has API ([old](https://developer.apple.com/documentation/appkit/nsaccessibility-swift.struct/attribute/verticalscrollbar?language=objc) and [new](https://developer.apple.com/documentation/appkit/nsaccessibility-c.protocol/accessibilityverticalscrollbar?language=objc)) through which an AT can obtain the scrollbars associated with a scrollview. While ARIA lacks a "scrollview" role, the [spec states](https://w3c.github.io/aria/#scrollbar):

> Authors MUST set the [aria-controls](https://w3c.github.io/aria/#aria-controls) attribute on the scrollbar element to reference the scrollable area it controls.

If authors do that, then user agents could implement NSAccessibility's scroll-bar API exposing the scrollbars on the controlled element.

My question is: Should user agents do this implementation, or was it deliberately left out of the spec?

@cookiecrook: Your/Apple's input would be much appreciated. Thanks in advance!

**Comment by spectranaut** [comment]
Discussed in triage in today's meeting, decision: yes -- I'm going to open a PR right now!

**Comment by cookiecrook** [comment]
> My question is: Should user agents do this implementation?

Yes.

> or was it deliberately left out of the spec?

No. Not intentional. Thanks.
